plugins/woocommerce-admin/client/products/fields/attribute-control/new-attribute-modal.tsx

```diff
@@ -80,15 +80,6 @@ export const AddAttributeModal: React.FC< AddAttributeModalProps > = ( {
 	onAdd,
 	selectedAttributeIds = [],
 } ) => {
-	const scrollAttributeIntoView = ( index: number ) => {
```

Comment:
This functionality needs to be put back in!

Reply:
🤦  Don't even recall removing this.  I wonder if I dunced a rebase or something.

Reply:
Rebasing... I'm always nervous I'm going to remove something accidentally!